### ISSUE TITLE
Stop presentation from restarting when focus is changed?

### ISSUE THREAD

**Issue by mimavox** [evilz/vscode-reveal#616]
Presentation shows fine in VS Code side view, but whenever I change focus to another tab the presentation restarts from the beginning. This makes it impossible to use this in my teaching. I would like to have a presentation going on the right-hand side while doing coding examples in Python files on the left. But whenever I try to do this, the presentation restarts from the beginning as soon as i change tab focus. Any way to prevent this?

**Comment by fuhrmanator** [comment]
The workaround for me is to view the presentation in a separate browser window outside of VSCode. If you want to keep the window inside VSCode, maybe you can use https://marketplace.visualstudio.com/items?itemName=auchenberg.vscode-browser-preview&ssr=false#review-details (it's not a perfect plug-in, but might be adequate for your needs). Point it to the URL generated by vscode-reveal (the localhost url).

**Comment by mimavox** [comment]
Yes, I'm using an external browser as well. It kind of defeats the purpose of having a preview plugin in VS Code though. I will check out the browser preview plugin, thank you.

**Comment by evilz** [comment]
@fuhrmanator Is right, Using it in VS code is focus on editing, if you show something else maybe you should export presentation first.

**Comment by evilz** [comment]
TODO :  Add option to stop sync preview

**Comment by evilz** [comment]
The preview is now a persistent VS Code webview. Moving focus to a non-Markdown editor does not refresh the active presentation, so it no longer restarts from slide one. This guard is covered by `src/test/UnitTests/MainController.jest.ts` (the non-Markdown editor event test).

Closing as completed in the current version.
